### PR TITLE
issue-14 add prettier dependency and config.

### PR DESCRIPTION
**Pull request by konstantinoff**

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "trailingComma": "es5",
```
Comment (ikachura) [inline .prettierrc:2]: have you tried parameter `all`? Is it change a lot of code while you reformat the code with this rule?

File: .prettierrc [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  "tabWidth": 2,
+  "semi": true,
+  "singleQuote": true,
+  "bracketSpacing": false,
+  "printWidth": 160,
+  "overrides": [
```
Comment (ikachura) [inline .prettierrc:8]: I think it's not relevant in our case. Can you please try to format the code without this param?

**Comment by ikachura** [comment]
@konstantinoff, I've added a PR with editorconfig. I think it might be enough now.
https://github.com/wix-incubator/rehabjs/pull/23

File: .prettierrc (hunk 2 above)
Comment (konstantinoff) [reply .prettierrc:8]: There is four params in code snippet. What exact param do you mean?

**Pull request merged**